viz/cmd/root.go

```diff
@@ -29,6 +36,10 @@ var (
 	alphaNumDash = regexp.MustCompile(`^[a-zA-Z0-9-]+$`)
 )
 
+func init() {
+	defaultNamespace = cmd.GetDefaultNamespace(kubeconfigPath, kubeContext)
```

Comment:
I think this has the same issue as https://github.com/linkerd/linkerd2/issues/5524 where `kubeconfigPath` and `kubeContext` will be empty when `init()` is called because the flags have not been parsed yet.